frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -197,8 +197,8 @@ export const NavBar: React.FC<NavBarProps> = ({ pages }) => {
     <div>
       <Tabs
         activeKey={defaultPage ? '' : lastElement}
-        inset={{ default: 'insetNone', xl: 'insetSm' }}
         component="nav"
+        className="co-horizontal-nav"
```

Comment:
To align the text in the Tabs component with the parent PageSection

```suggestion
        className="pf-v6-u-mx-xs"
```

Reply:
Still needed for the tab inset fix.